js/ui/calendar/ui.calendar.js

```diff
@@ -709,11 +764,30 @@ const Calendar = Editor.inherit({
             this._navigateDown(e.event.currentTarget);
         } else {
             const newValue = this._updateTimeComponent(e.value);
-            this._dateValue(newValue, e.event);
+            this._selectValue(newValue, e.event);
             this._cellClickAction(e);
         }
     },
 
+    _selectValue: function(selectedValue, e) {
+        if(this.option('multiselect')) {
+            const values = [...this._dateOption('values')];
+            const alreadySelectedIndex = values.findIndex(date => date.toDateString() === selectedValue.toDateString());
+
+            if(alreadySelectedIndex > -1) {
+                values.splice(alreadySelectedIndex, 1);
+            } else {
+                values.push(selectedValue);
+            }
+
+            this._currentDateChanged = true;
+            this.option('currentDate', isDefined(selectedValue) ? selectedValue : new Date());
```

Comment:
We can refactor this way:
```suggestion
            this.option('currentDate', selectedValue ?? new Date();
```
But y do we need set `new Date()`?

Reply:
it was before my changes. Why? Because some cell should be focused even if there is no selected value, today's date seems good to focused